pkg/cmd/util/sibling.go

```diff
@@ -26,5 +34,21 @@ func SiblingCommand(cmd *cobra.Command, name string) string {
 	command = append(command, name)
 	glog.V(4).Infof("The sibling command is: %s", strings.Join(command, " "))
 
-	return strings.Join(command, " ")
+	return command
+}
+
+// EscapeCommand wraps each element of the command in double quotes
+// if it contains any of the following: single quote, double quote, backslash, or space
+// It also replaces every pre-existing quoting character in the element with a pair.
+// Examples: ' -> '', " -> ""
+func EscapeCommand(command []string) []string {
+	for key := range command {
+		needsQuoted := MatchEscapeCharacters.MatchString(command[key])
```

Comment:
- use strings.ContainsAny and strings.Replace, not regexes
- you've chosen to quote using " so replacement ' -> '' must not be done (but you still need to quote if the string contains a ')
- guard replace " -> "" with needsQuoted
- command[key] = \`"\` + command[key] + \`"\` saves the fmt.Sprintf and raw quotes are clearer here